.circleci/config.yml

```diff
@@ -12,9 +12,10 @@ jobs:
           SLAPD_PASSWORD: 'redbrick'
           SLAPD_DOMAIN: 'ldap.internal'
     steps:
-      - run: apk add --no-cache openldap-dev cyrus-sasl-dev git build-base
+      - run: apk add --no-cache openldap-dev openldap-clients cyrus-sasl-dev git build-base
       - checkout
       - run: pip install -r useradm/requirements.txt
+      - run: ldapsearch -x -h ldap.internal -b "ou=accounts,o=redbrick"
```

Comment:
thast just setting the domain name of the server in ldap for the schema not actually creating a netowork